src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -134,6 +167,9 @@ const RenderListFeed = (props) => {
             onCardContentPress={() => navigateToLinkContextPage(item)}
             message={item.message}
             topics={item?.topics}
+            contentHeight={
+              dimen.size.FEED_CURRENT_ITEM_HEIGHT - getHeightHeader() - getHeightReaction()
+            }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `contentHeight` prop has been added to the `Content` component. Ensure that this prop is handled correctly in the `Content` component.
```diff
+            contentHeight={
+              dimen.size.FEED_CURRENT_ITEM_HEIGHT - getHeightHeader() - getHeightReaction()
+            }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->